static/app/components/events/interfaces/nel/help/index.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled';
+
+import ExternalLink from 'sentry/components/links/externalLink';
+import {IconOpen} from 'sentry/icons';
+import {space} from 'sentry/styles/space';
+
+import nelProperties from './nelProperties';
+
+function NELHelp({data}) {
+  const getLink = () => {
+    const href =
+      'https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging#error_reports';
+
+    return (
+      <StyledExternalLink href={href}>
+        {'developer.mozilla.org'}
+        <IconOpen size="xs" className="external-icon" />
+      </StyledExternalLink>
```

Comment:
nit: Why does it need to be in a separate function. IMHO this simply could be inlined in the main JSX.